server/adapters/adapter.go

```diff
@@ -10,6 +10,7 @@ type SQLAdapter interface {
 	PatchTableSchema(schemaToAdd *Table) error
 	BulkInsert(table *Table, objects []map[string]interface{}) error
 	BulkUpdate(table *Table, objects []map[string]interface{}, deleteConditions *DeleteConditions) error
+	Clean(tableName string) error
```

Comment:
It would be better to have more clear name e.g. `Truncate`

Reply:
renamed method